src/Database/Database.php

```
@@ -921,14 +921,25 @@ public function deleteDocument(string $collection, string $id): bool
      * @param int $offset
      * @param array $orderAttributes
      * @param array $orderTypes
+     * @param string $orderAfter
      *
      * @return Document[]
      */
-    public function find(string $collection, array $queries = [], int $limit = 25, int $offset = 0, array $orderAttributes = [], array $orderTypes = []): array
+    public function find(string $collection, array $queries = [], int $limit = 25, int $offset = 0, array $orderAttributes = [], array $orderTypes = [], string $orderAfter = null): array
```

It would be hard to return a proper error to the HTTP response. We can either pass a document instance and validate it in the controller before we pass it here, or we can throw a custom exception and make sure we catch it in the relevant route and delegate a proper HTTP error.

---

Fixed 👍🏻